tests/e2e/configurer/chain/chain.go

```diff
@@ -43,6 +53,8 @@ const (
 	waitUntilRepeatPauseTime = 2 * time.Second
 	// waitUntilrepeatMax is the maximum number of times to repeat the wait until condition.
 	waitUntilrepeatMax = 60
+
+	proposalStatusPassed = "PROPOSAL_STATUS_PASSED"
```

Comment:
This seems unrelated to this PR. Do we want it here?

Reply:
This was used to ensure that the param change proposal passed before we tried to get the param itself. This could help to be more confident that the proposal not passing was the reason for the parameter not updating. However, that check seems to be removed now.

I'm cool with not blocking on this and just deleting `prposalStatusPassed`

Reply:
Ahh, actually it's still being used for checking that the proposal passed here: https://github.com/osmosis-labs/osmosis/blob/376fccde469c3c4168127e48ee9beda44cee7647/tests/e2e/configurer/chain/chain.go#L285

So I think we can keep this

Reply:
great!